contentctl/actions/detection_testing/views/DetectionTestingView.py

```diff
@@ -143,33 +171,43 @@ def getSummaryObject(
         # Compute total detections
         total_detections = total_fail + total_pass + total_untested + total_skipped
 
+        # Compute total detections actually tested (at least one test not skipped)
+        total_tested_detections = total_fail + total_pass
 
         # Compute the percentage of completion for testing, as well as the success rate
         percent_complete = Utils.getPercent(
             len(tested_detections), len(untested_detections), 1
         )
         success_rate = Utils.getPercent(
-            total_pass, total_detections-total_skipped, 1
+            total_pass, total_tested_detections, 1
         )
 
-        # TODO (#230): expand testing metrics reported
+        # TODO (cmcginley): the dual language of skipped_detections and untested_detections is a
+        #   bit confusing; consider renaming
+        # TODO (#230): expand testing metrics reported (and make nested)
         # Construct and return the larger results dict
         result_dict = {
+            # TODO (cmcginley): differentiate total detections vs total tested
             "summary": {
+                "mode": self.config.getModeName(),
+                "enable_integration_testing": self.config.enable_integration_testing,
                 "success": overall_success,
                 "total_detections": total_detections,
+                "total_tested_detections": total_tested_detections,
                 "total_pass": total_pass,
                 "total_fail": total_fail,
                 "total_skipped": total_skipped,
                 "total_untested": total_untested,
-                "total_experimental_or_deprecated": len(deprecated_detections+experimental_detections),
+                "total_production": total_production,
+                "total_experimental": total_experimental,
+                "total_deprecated": total_deprecated,
+                "total_manual": total_manual,
+                "total_other_skips": total_skipped - total_deprecated - total_experimental - total_manual,
                 "success_rate": success_rate,
             },
             "tested_detections": tested_detections,
+            "skipped_detections": skipped_detections,
             "untested_detections": untested_detections,
```

Comment:
Do we have any thoughts about the language around tested, untested and skipped detections? I feel like the lines between untested and skipped get blurry as it relates to tested detections

Reply:
I like the idea of better aligning everything w/ the status enums. 

I already had a desire to add an additional enum `UNSET` or something similar to demonstrate a test which has not yet been tested but is intended to be. I want to do this primarily bc I don't like allowing for `None` in the `status` field, but I think it helps in this case too. So what if we:

1. Eliminated `total_untested` and `untested_detections`
2. If testing fails in an unhandled way that causes contentctl to stop altogether, then we iterate over all tests, and for any that remain `UNSET` after testing completes, we mark the status as `ERROR` with a message like `TEST FAILED (ERROR): testing end state failed to be set, testing may have ended prematurely`

Reply:
Yes, I agree with this. I will leave this open (instead of resolving) as I'm not sure if you want to make that change in THIS PR vs opening an issue where this is future work.

Reply:
Gonna capture this as an issue for future work to avoid scope creep :)

Reply:
https://github.com/splunk/contentctl/issues/267

Added TODOs in most recent commits